DependencyInjection/SuluFormExtension.php

```diff
@@ -72,7 +81,7 @@ public function load(array $configs, ContainerBuilder $container)
         // add dynamic lists
         foreach ($config['dynamic_lists'] as $key => $value) {
             $parameter = 'sulu_form.dynamic_lists.' . $key . '.config';
-            $container->setParameter($parameter, $value);
+            $container->setParameter($parameter, $value ?: []);
 
             $definition = new Definition(DynamicListNavigationProvider::class);
             $definition->addArgument('%' . $parameter . '%');
```

Comment:
type should also not be required:

so we can also add this to dynamic_lists to show all result of a form:

dynamic_lists:
    sulu_form_form:
        result:
              property: id